src/arizona_template_compiler.erl

```diff
@@ -185,8 +191,13 @@ compile_block(#{name := Name} = Block, Loc, T, State) ->
     {Mod, Fun} = block_mod_fun(Name, Loc, State),
     case erlang:function_exported(Mod, Fun, 1) of
         true ->
-            Attrs = maps:get(attributes, Block),
-            Elems = expand_block(Mod, Fun, Attrs, State),
+            Elems = case expand_is_visible(maps:get(is_visible, Block), State) of
```

Comment:
The old "case of case" :) I'll write up an issue so we get rid of as much of these as we can.

Reply:
https://github.com/arizona-framework/arizona/issues/188

(feel free to resolve this conversation to move the pull request forward)